api/types/types.go

```diff
@@ -122,6 +122,7 @@ const (
 	SiteCaSecret             string = "skupper-site-ca"
 	ConsoleServerSecret      string = "skupper-console-certs"
 	OauthRouterConsoleSecret string = "skupper-router-console-certs"
+	SiteCaServicesSecret     string = "skupper-site-ca-services"
```

Comment:
Minor issue but I think the name could be 'skupper-service-ca' (that aligns well with the other two existing cas, skupper-local-ca and skupper-site-ca.

Reply:
Done!